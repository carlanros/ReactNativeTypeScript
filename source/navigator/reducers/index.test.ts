// navigator/reducers/index.test
/// <reference types="jest"/>
import AppNavReducer from './index';

describe('AuthReducer', () => {
  test('should return initial state', () => {
    expect(
      AppNavReducer(undefined, { type: null, scene: {} })
    ).toMatchSnapshot();
  });
  const mockScene = { path: '/home' };

  test(`should handle focus`, () => {
    expect(
      AppNavReducer(undefined, { type: 'focus', scene: mockScene })
    ).toEqual({
      scene: mockScene,
    });
  });
});
