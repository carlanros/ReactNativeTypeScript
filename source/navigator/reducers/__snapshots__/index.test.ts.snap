// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthReducer should return initial state 1`] = `
Object {
  "scene": Object {},
}
`;
